native/cocos/editor-support/spine-wasm/spine-skeleton-instance.cpp

```diff
@@ -11,14 +11,24 @@ SlotMesh globalMesh(nullptr, nullptr, 0, 0);
 
 extern "C" {
 extern void spineListenerCallBackFromJS();
+extern void spineTrackListenerCallback();
 }
 using namespace spine;
 
 static void animationCallback(AnimationState *state, EventType type, TrackEntry *entry, Event *event) {
-    SpineSkeletonInstance *instance = (SpineSkeletonInstance *)state->getRendererObject();
+    SpineSkeletonInstance *instance = (static_cast<SpineSkeletonInstance *>(state->getRendererObject()));
     instance->onAnimationStateEvent(entry, type, event);
 }
 
+void trackEntryCallback(AnimationState *state, EventType type, TrackEntry *entry, Event *event) {
+    (static_cast<SpineSkeletonInstance *>(state->getRendererObject()))->onTrackEntryEvent(entry, type, event);
+    if (type == EventType_Dispose) {
+        if (entry->getRendererObject()) {
```

Comment:
Would the renderobject be nullptr? If so, the above line 
```
(static_cast<SpineSkeletonInstance *>(state->getRendererObject()))->onTrackEntryEvent(entry, type, event);
```
should also consider the situation of `nullptr`.